pkg/app/application.go

```diff
@@ -61,7 +62,12 @@ func NewApplication(ldr loader.Loader, fSys fs.FileSystem) (*Application, error)
 	if err != nil {
 		return nil, err
 	}
-
+	if m.PatchesJson6902 != nil {
+		log.Printf("field patchesJson6902 ignored; no implementation yet.")
+	}
+	if m.Patches != nil {
+		log.Println("field patches will be deprecated, please change it to patchesStategicMerge")
```

Comment:
Let's not warn yet.

First let's change all the docs and examples to use the new field ASAP.  Then file an issue to deprecate `patches`.  No rush.